glean-core/src/glean.udl

```diff
@@ -69,6 +69,17 @@ dictionary InternalConfiguration {
     string app_build;
     boolean use_core_mps;
     boolean trim_data_to_registered_pings;
+    LevelFilter? log_level;
+};
+
+// An enum representing the different logging levels for the `log` crate.
+enum LevelFilter {
+    "Debug",
```

Comment:
The ordering here seems a bit weird.
Probably best to keep it in the order of increasing log level, e.g. see how the Rust crate defines it: https://docs.rs/log/0.4.17/log/enum.Level.html